flycheck.el

```diff
@@ -3942,8 +3967,10 @@ Return a list with the contents of the table cell."
                    column 'flycheck-error-list-column-number)
                   (flycheck-error-list-make-cell
                    (symbol-name (flycheck-error-level error)) level-face)
-                  (flycheck-error-list-make-cell
-                   id-str 'flycheck-error-list-id id-str)
+                  ;; Error ID has a button to explain the error
+                  (list id-str
+                        'type 'flycheck-error-list-explain-error
```

Comment:
In these instances it would be useful to be able to insert pictures in the review ;)

In any case, I've added a different face for error-id when an explainer is present.  I've used a released-button box style, but an hyper link might be good as well.  If it's too much, I can revert the commit as well.